### ISSUE TITLE
temps de chargement des pages long

### ISSUE THREAD

**Issue by ffameli** [cafebabel/cafebabel.com#521]

**Comment by ffameli** [comment]
test : 2"58 de chargement sur cafebabel.com / 3"25 sur slate. 
remarque : pas de moulin sur slate, ni rideau blanc
proposition : enlever le rideau et laisser s'affciher la page sans le rideau
faire ce test pour voir si nous avons la même sensation
@cafebabel-webmaster peux tu essayer ?

**Comment by johangiraud** [comment]
Non. Ça fait partie de l'identité du site, ce n'est pas ça qui ralentit le site, au contraire ça permet de savoir que la page charge plutôt que d'attendre devant une page immobile sans comprendre pourquoi.

**Comment by matthieuamare** [comment]
Perso, j'ai plutôt 6'' sur cafebabel.com, parfois 10'' sur la version mobile.

**Comment by ffameli** [comment]
tu dois avoir un outil dans ton navigateur pour avoir le temps exact de chargement. Johan pourra te le montrer le cas échéant une fois qu'il aura bossé avec Vincent sur le test.
